models/User.go

```diff
@@ -1445,7 +1445,7 @@ func PerformMortgageTransaction(userId, stockId uint32, stockQuantity int32) (*T
 		MortgagePutLimitRWMutex.RLock()
 		lim := MortgagePutLimit
 		MortgagePutLimitRWMutex.RUnlock()
-		if user.Total < lim {
+		if user.Total > lim {
```

Comment:
`lim` is a constant set to 4000000. So the user cannot mortgage if he has networth above the limit, since he does not need to mortgage right now. So I guess `user.Total > lim` should throw an error instead of `user.Total < lim`. @gauthamk97

Reply:
Yeah. There should've been a test which made sure a ```WayTooMuchCashError``` was thrown when user.Total was greater than 4000000. Would've caught this mistake.